library/network/test/susefirewalld_services_test.rb

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env rspec
+
+require_relative "test_helper"
+
+Yast.import "SuSEFirewallServices"
+Yast.import "SCR"
+Yast.import "FileUtils"
+
+describe Yast::SuSEFirewallServices do
+
+  before :example do
+    # We shouldn't run this test if firewalld is running
+    if Yast::SuSEFirewallServices.is_a?(Yast::SuSEFirewall2ServicesClass)
+      skip "SuSEfirewall2 backend is not supported by this test" do
```

Comment:
I don't have a ready answer for the tests either, yet.
For the failures, please log in to dhcp112.suse.cz (credentials elsewhere) , `cd /root/yast-yast2` and `rake test:unit`.